lib/livebook_web/live/settings_live.ex

```diff
@@ -314,10 +317,26 @@ defmodule LivebookWeb.SettingsLive do
 
   def handle_event("detach_file_system", %{"id" => file_system_id}, socket) do
     Livebook.Settings.remove_file_system(file_system_id)
+
+    case Ets.fetch(:file_system, "default_file_system") do
```

Comment:
We should also have a function for this in settings, and if no default file system is set, it should return the local file system (which is the implicit default) :)